src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  src: url("../src/fonts/Avenir.ttc") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("../src/fonts/Avenir-Book.ttf") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("../src/fonts/Avenir-Heavy.ttf") format("opentype");
+}
+
+form {
+  margin: 20px 0;
+}
+
+.searchBig input {
```

Comment:
Don't use rules for complex selectors with tags. Just add class name for `input` and move these rules to the class selector. For example:
```
.search__input {
  ...
}
```